src/upload-tests.proj

```diff
@@ -77,10 +77,10 @@
     <HelixArchLabel>$(ArchGroup)</HelixArchLabel>
     <HelixConfigLabel>$(ConfigurationGroup)</HelixConfigLabel>
     <MaxRetryCount>1</MaxRetryCount>
-    <!-- Unique container name per build and configuration, e.g., corefx-beta-12345-01-windowsnt-x64-netcoreapp-master -->
+    <!-- Unique container name and blob prefix per build and configuration, e.g., corefx-beta-12345-01/tests-master-windowsnt-x64-netcoreapp-master -->
     <!-- Azure Storage containers can only contain [a-z, 0-9, -] in their names and $(OSGroup) is "Windows_NT" on Windows... -->
-    <RawContainerName Condition="'$(Branch)' != ''">corefx-$(Branch)-$(PreReleaseLabel)-$(BuildNumberMajor)-$(BuildNumberMinor)-$(OSGroup)-$(ArchGroup)-$(TargetGroup)</RawContainerName>
-    <ContainerName Condition="'$(ContainerName)' == '' And '$(IsOfficial)' == 'true' And '$(Branch)' != ''">$([System.Text.RegularExpressions.Regex]::Replace('$(RawContainerName.ToLower())', "[^A-Za-z0-9-]+", ""))</ContainerName>
+    <ContainerName Condition="'$(ContainerName)' == '' And '$(IsOfficial)' == 'true'">corefx-$(PreReleaseLabel)-$(BuildNumberMajor)-$(BuildNumberMinor)</ContainerName>
```

Comment:
Since we are moving to Blob prefix than I think we should keep a stable container name with just "corefx-tests" or something like that move all the changing parameters into blob relative paths.

Reply:
That would move _all_ offcial builds into 1 storage container with a directory inside it for every configuration x branch x build orchestration.  This currently puts all the testing artifacts of 1 orchestrated official build into a single storage container.  Would you rather it be 
```
corefx-test/$(prerelease)-$(major)-$(minor)-$(branch)-$(OS)-$(Arch)-$(Target)
```
?

Reply:
Yes something like that is what I was looking for. You could also make some of the other things directories like:

`corefx-tests/$(branch)/$(prerelease)-$(major)-$(minor)-$(OS)-$(Arch)-$(Target)`